test/strategy/EnrichClientFromRemoteConfigStrategyTest.js

```diff
@@ -0,0 +1,135 @@
+'use strict';
+
+var stormpath = require('stormpath');
+
+var common = require('../common');
+var _ = common._;
+var assert = common.assert;
+var sinon = common.sinon;
+
+var strategy = require('../../lib/strategy');
+var EnrichClientFromRemoteConfigStrategy = strategy.EnrichClientFromRemoteConfigStrategy;
+
+describe('EnrichClientFromRemoteConfigStrategy', function () {
+  var client, mockApplications, testStrategy;
+
+  before(function (done) {
+    mockApplications = [{
+      href: 'https://stormpath.mock/applications/stormpath',
+      name: 'Stormpath'
+    }, {
+      href: 'https://stormpath.mock/applications/my-application',
+      name: 'My Application'
+    }];
+
+    client = new stormpath.Client({
+      skipRemoteConfig: true
+    });
+
+    testStrategy = new EnrichClientFromRemoteConfigStrategy(function () {
+      return client;
+    });
+
+    client.on('error', function (err) {
+      throw err;
+    });
+
+    client.on('ready', function () {
+      done();
+    });
+  });
+
+  it("should resolve application by name", function (done) {
```

Comment:
Okay, but why are you then using single quotes everywhere else in this file except for `it` statements?